5.4.kalim_shlovim.py

```diff
@@ -0,0 +1,44 @@
+SPACE = ' '
+
+
+def interleaveGenerator(*args):
```

Comment:
Python is snake case -
[https://www.google.com/amp/s/www.theserverside.com/answer/Camel-case-vs-snake-case-Whats-the-difference%3famp=1](https://www.google.com/amp/s/www.theserverside.com/answer/Camel-case-vs-snake-case-Whats-the-difference%3famp=1?authuser=0)